openlibrary/plugins/openlibrary/js/covers.js

```diff
@@ -27,14 +27,14 @@ export function initCoversChange() {
                 add_iframe('#imagesManage', manage_url);
             }, 0);
         })
-        .bind('cbox_cleanup', function () {
+        .on('cbox_cleanup', function () {
             $('#imagesAdd').html('');
             $('#imagesManage').html('');
         });
 }
 
 function val(selector) {
-    return $.trim($(selector).val());
+    return ($(selector).trim().val());
```

Comment:
No need for the surrounding brackets here.

Reply:
Also this needs to apply to the result of val, otherwise this is not the same end result.